src/System.Net.Http/tests/StressTests/HttpStress/Program.cs

```diff
@@ -18,30 +19,31 @@
 public static class Program
 {
 
-    public static async Task Main(string[] args)
+    public static async Task<int> Main(string[] args)
     {
-        if (!TryParseCli(args, out Configuration config))
+        if (!TryParseCli(args, out Configuration? config))
         {
-            return;
+            return 2;
         }
 
-        await Run(config);
+        return await Run(config);
     }
 
-    private static bool TryParseCli(string[] args, out Configuration config)
+    private static bool TryParseCli(string[] args, [NotNullWhen(true)] out Configuration? config)
     {
         var cmd = new RootCommand();
         cmd.AddOption(new Option("-n", "Max number of requests to make concurrently.") { Argument = new Argument<int>("numWorkers", Environment.ProcessorCount) });
         cmd.AddOption(new Option("-serverUri", "Stress suite server uri.") { Argument = new Argument<Uri>("serverUri", new Uri("https://localhost:5001")) });
```

Comment:
It might be useful to modify the help text to show that this parameter has a default, `http://localhost:5001/`

Comment applies to other parameters that have defaults now.

Reply:
The value serves to convince the compiler that ServerUri cannot be null. The cli processor will always populate with a default value of its own which is documented. This addition saves us from a bunch of `!` operators in the code.

Perhaps I could name this `http://just-a-placeholder` to avoid confusion. Or replace setters with constructor parameters.